client/src/components/ui/ImageCard.style.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+interface SizeOfImageCard {
+  img: string;
+  imageSize?: {
+    width: string;
+    height: string;
+  };
+  mobileImageSize?: {
+    width: string;
+    height: string;
+  };
+}
+
+export const ImageCard = styled.div<SizeOfImageCard>`
```

Comment:
where do you use mobile Image Size? I wonder if the img tag would be better to handle images, where can you also enter alt attribute?